packages/core/src/sdJwt/disclosures.ts

```diff
@@ -2,11 +2,14 @@ import { DisclosureItem, Hasher } from '../types'
 import { Base64url } from '../base64url'
 import { SdJwtError } from './error'
 
+// Make the digest property required
+export type DisclosureWithDigest = Disclosure & { digest: string }
+
 export class Disclosure {
     private salt: string
     private key?: string
     private value: unknown
-    private _digest: string | undefined
+    #digest?: string
```

Comment:
This is something I really have to get used to. Is there a specific reason why this would be marked as a private property on a js level and the others on a ts level?

Reply:
# makes the property actually private, while ts just hides it based on the type (so you can still access it).

The reason i used this is because it automatically makes it private and it doesn't clash with the 'digest' name.

But if you want i can also revert it to _digest. Or even `public digest` I think

Reply:
No, it's fine, it makes sense.